src/Propel/Runtime/Connection/ProfilerConnectionWrapper.php

```diff
@@ -69,9 +70,9 @@ public function setAttribute($attribute, $value)
                 // Set whether the connection must only log slow queries.
                 // The slow threshold must be set on the profiler (100ms by default).
                 $this->isSlowOnly = $value;
-                break;
+                return true;
             default:
-                parent::setAttribute($attribute, $value);
+                return parent::setAttribute($attribute, $value);
```

Comment:
Looks like the returns are changed here in unexpected ways. Both cases would have returned null, but now it returns true or forwards a parent's value.

Reply:
I dont think results are used. Contracts say bool though.

Reply:
also fine